app/views/guidance/selecting-sites-for-development-for-your-local-plan/stage-2-categorising-and-assessing-sites.html

```diff
@@ -0,0 +1,768 @@
+                      constraints that mean it might not be suitable land to
+                      build on.&nbsp;
+                    </p>
+                    <p>
+                      If any of your sites are affected by these constraints,
+                      you should ideally discard those sites first, as they are
+                      the hardest to mitigate. This will be different depending
+                      on the area and some of these constraints might not stop
+                      you from taking your sites forward into a detailed
+                      assessment. You should use your judgement based on your
+                      local area and the sites you have available.&nbsp;
+                    </p>
+                    <p>
+                      There are a number of national constraints that could
+                      affect your ability to build on sites, which includes:
+                    </p>
+                    <ul>
+                      <li>
+                        Ramsar/Special Protection Area/Special Areas of
+                        Conservation
+                      </li>
+                      <li>Sites of Special Scientific Interest (SSSIs)</li>
+                      <li>Scheduled Monuments</li>
+                      <li>Ancient Woodlands (primarily)</li>
+                      <li>Flood Zones 3a and 3b</li>
+                      <li>Public Safety Zones (i.e. airports)</li>
+                      <li>Detailed Emergency Planning Zones</li>
+                      <li>Health &amp; Safety Executive Consultation Zones</li>
+                      <li>Village Green / Common Land</li>
+                    </ul>
+                    <p>
+                      You should use your judgement based on your local area and
+                      the sites you have available to decide if these
+                      constraints mean you cannot develop on the site. If you
+                      find any of your sites have these constraints, but you can
+                      mitigate the risks, you should record this and take the
+                      sites forward for detailed assessment.
+                    </p>
+                    ' } }, { heading: { text: "Assessing the suitability,
+                    availability and achievability of sites" }, content: { html:
+                    '
+                    <h3 id="links-2">Links</h3>
+                    <p>
+                      <a href="">Assessing the suitability of sites</a>
+                    </p>
+                    <p><a href="">Constraints to consider</a></p>
+                    <p>
+                      <a href="">Using our approach to score your sites</a>
+                    </p>
+                    <p>
+                      <a href="">Assessing the availability of sites</a>
+                    </p>
+                    <p>
+                      <a href="">Assessing the achievability of sites</a>
+                    </p>
```

Comment:
It looks like these links (and the other ones I've commented on) need to be hash links (e.g. `#assessing-the-suitability-of-sites`)